src/obu.rs

```diff
@@ -704,8 +704,8 @@ unsafe fn parse_refidx(
                         .frame_hdr
                         .as_ref()
                         .ok_or(EINVAL)?
-                        .frame_offset,
-                    frame_offset,
+                        .frame_offset as c_int,
+                    frame_offset as c_int,
```

Comment:
We could make `fn get_poc_diff` and all of the `poc` fields/vars `u8`s.  I can add that if you want (I checked to see if it it's possible).

Reply:
Yes, but that's really beyond the scope of this PR. I'll let you create a new one.